### PR TITLE
Fix `Last Balance Check` checks

### PR DESCRIPTION
**Pull request by trizin**
Changes proposed in this PR:

- Fix `Last Balance Check` checks
- Remove balance check in `prepareProposalsForSnapshot` function, this causes the balances to be checked twice. It is called in `prepareNewProposals` function in `update_funding_round.js` file.

### CODE REVIEW

File: src/airtable/proposals/proposal_standings.js [hunk 1]
```diff
@@ -185,13 +185,19 @@ const getProposalRecord = async (proposal, allProposals, curRoundNumber) => {
...
-  let areOceansEnough = proposal.get('Proposal Standing') !== 'No Ocean'
+  let areOceansEnough =
+    proposal.get('Proposal Standing') !== '' &&
+    proposal.get('Proposal Standing') !== 'No Ocean'
   if (
-    !(curRoundNumber !== undefined && curRoundNumber !== proposal.get('Round'))
+    !(
+      curRoundNumber !== undefined &&
+      parseInt(curRoundNumber) !== parseInt(proposal.get('Round'))
+    )
```
Comment (idiom-bytes) [inline src/airtable/proposals/proposal_standings.js:195]: Added checks make sense.

**Comment by idiom-bytes** [comment]
Updates make sense. Am just wondering if there are edge cases around it that should be getting tested.

**Comment by trizin** [comment]
Feel free to test it on your local! I believe I tested most of the cases. @idiom-bytes

**Comment by trizin** [comment]
We should reconsider this change:
https://github.com/oceanprotocol/DAOBot/pull/164/files#diff-d08d59ff6fd7a3061d544f05bc7122a70312ee9614e5b24cd07ffaede2c63a09L56-L67

**Review by KatunaNorbert — APPROVED** [review]
It looks good

File: src/airtable/proposals/proposal_standings.js [hunk 2]
```diff
@@ -185,13 +188,19 @@ const getProposalRecord = async (proposal, allProposals, curRoundNumber) => {
...
   ) {
     if (
       !lastOceanBalanceCheckDate ||
-      new Date(lastOceanBalanceCheckDate) + 1000 * 60 * 15 < new Date() // undefined or 15 minutes has passed
+      new Date(lastOceanBalanceCheckDate).getTime() + 1000 * 60 * 15 <
+        Date.now() // undefined or 15 minutes has passed
```
Comment (idiom-bytes) [inline src/airtable/proposals/proposal_standings.js:203]: Which linter is recommending this change?
Does this break `npm run lint`?

File: src/snapshot/prepare_snapshot_received_proposals_airtable.js [hunk 3]
```diff
@@ -53,7 +50,6 @@ const prepareProposalsForSnapshot = async (curRound) => {
     proposals.map(async (proposal) => {
       getProposalRecord(proposal, proposals)
       try {
-        const wallet_0x = proposal.get('Wallet Address')
         const proposalStanding = proposal.get('Proposal Standing')
```
Comment (idiom-bytes) [inline src/snapshot/prepare_snapshot_received_proposals_airtable.js:53]: I am wondering how you went about testing this update.

**Comment by idiom-bytes** [comment]
Note: Test execution time has nearly doubled from 2m => 4m.

File: src/snapshot/prepare_snapshot_received_proposals_airtable.js (hunk 3 above)
Comment (trizin) [reply src/snapshot/prepare_snapshot_received_proposals_airtable.js]: Rolling back this change will cause DAOBot to check all Polygon, BSC and Ethereum every minute and before voting begins. A failure in one of the RPCs - which is common - can cause some problems. @idiom-bytes

File: src/airtable/proposals/proposal_standings.js (hunk 2 above)
Comment (trizin) [reply src/airtable/proposals/proposal_standings.js:203]: Yes
